book_source/03_intermediate_users_guide/01_pecan_xml.Rmd

```diff
@@ -1,195 +1,304 @@
 # The PEcAn XML {#pecanXML}
 
-The PEcAn system is configured using a xml file, often called settings.xml. The configuration file contains the following sections:
-1. [Info]
-2. [PEcAn Folders]
-3. [Database Access]
-4. [BETY Configuration](#bety-database-configuration)
-5. [Brown Dog](#brown-dog-configuration)
-6. [PFT Selection](#pft-selection)
-7. [Meta Analysis](#meta-analysis)
-8. [Ensemble Runs](#ensemble-runs)
-9. [Sensitivity Runs](#sensitivity-runs)
-10. [Model Setup](#model-setup)
-11. [Run Setup](#run-setup)
-12. [State Data Assimilation](#state-data-assimilation-tags)
-13. [Parameter Data Assimilation]
-14. [Benchmarking]
-
-## Info
-The information section, tagged `info`, contains information specific to your run. They are not critical settings, so any missing values will not break your run. 
-
-An example of this section looks like:
+The PEcAn system is configured using a XML file, often called `pecan.xml`.
+A basic example looks like this:
```

Comment:
For these larger sections, I've seen that people like these smaller table of contents. Even though the sections are denoted as 2nd level headers and will show up in the sidebar. It's design choice, but from what I've seen people like these small intro sections so they know what's on the page and can click to that section.

Reply:
Fair enough. I wish they were programmatically generated, rather than manually maintained, but the people want what they want.